Index: source/install.rdf.temp
===================================================================
RCS file: /cvs/linkvisitor/source/install.rdf.temp,v
retrieving revision 1.6
diff -u -3 -r1.6 install.rdf.temp
--- source/install.rdf.temp	16 Apr 2005 18:28:54 -0000	1.6
+++ source/install.rdf.temp	4 Mar 2007 12:48:42 -0000
@@ -14,7 +14,7 @@
       <Description>
         <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
         <em:minVersion>1.0</em:minVersion>
-        <em:maxVersion>1.0</em:maxVersion>
+        <em:maxVersion>2.0+</em:maxVersion>
       </Description>
     </em:targetApplication>
 
